src/components/PageMyList/index.js

```diff
@@ -201,6 +202,11 @@
             <title>DIMES: My List</title>
           </Helmet>
           <div className="container mylist flex">
+            <nav>
+              <a href="/search" className="btn btn--new-search">
+                <MaterialIcon icon="keyboard_arrow_left"/>Start a New Search
```

Comment:
I'm wondering if, since this starts a fresh search, it may make more sense to go to the homepage? This goes to a search results page with an undefined search (but that indicates there are matches), which may be misleading.

Reply:
Good call - I totally meant to do that originally...